content/docs/introduction/monitoring-page.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Monitoring dashboard
+enableTableOfContents: true
+---
+
+The **Monitoring** dashboard in the Neon console provides several graphs for monitoring system and database metrics. You can access the **Monitoring** dashboard from the sidebar in the Neon Console. Observable metrics include:
+
+- [RAM](#ram)
+- [CPU](#cpu)
+- [Connections count](#connections-count)
+- [Buffer cache hit rate](#buffer-cache-hit-rate)
+- [Database size](#database-size)
+- [Deadlocks](#deadlocks)
+- [Rows](#rows)
+
+Your Neon plan defines the range of data you can view.
+
+| Neon Plan | Data Access  |
+|-----------|--------------|
+| [Free Tier](/docs/introduction/plans#free-tier) | Last hour     |
+| [Launch](/docs/introduction/plans#launch)    | Last day (24 hours)       |
+| [Scale](/docs/introduction/plans#scale)     | Last 7 days      |
+
+The dashboard displays metrics for the selected **Branch** and **Compute endpoint**. Use the drop-down menus to view metrics for a different branch or compute endpoint. Use the **Refresh** button to update the displayed metrics.
+
+If there has not been much activity, charts may display this message: `There is not enough metrics data for this compute`. In this case, try again later after more usage data has been collected.
```

Comment:
This could also mean that their compute is/was suspended.

Reply:
Fixed.